src/event-schemas/meta-data.json

```diff
@@ -65,6 +65,18 @@
         },
         "pageTags": { "type": "array", "items": { "type": "string" } }
     },
+    "patternProperties": {
+        "^(?!pageTags)[a-zA-Z0-9_]{1,128}$": {
```

Comment:
question: Do we need these restrictions? Will these be enforced by the service?

Reply:
Don't need character restrictions but will want to keep length and type validations, which will be enforced by backend. Updated to remove char restrictions.